doc/util.md

```diff
@@ -4,38 +4,48 @@ On this page you can find documentation for some utility classes and methods.
 
 ## Device Class
 
-This class, accessible via `Flame.device`, has some functions that can be used to control the state of the device, like changing the screen orientation for example.
+This class, accessible via `Flame.device`, has some functions that can be used to control the state
+of the device, like changing the screen orientation for example.
```

Comment:
again like + for example. DRY ;)

Reply:
Haha DRY does not apply for natural languages, "like" + "for example" is very commonly used together.

Reply:
Rewrote the sentence anyways, flowed badly with all the commas.